scripts/hostcfgd

```diff
@@ -204,12 +206,18 @@ class FeatureHandler(object):
     FEATURE_STATE_DISABLED = "disabled"
     FEATURE_STATE_FAILED = "failed"
 
-    def __init__(self, config_db, feature_state_table, device_config):
+    def __init__(self, config_db, feature_state_table, device_config, is_advanced_boot):
         self._config_db = config_db
         self._feature_state_table = feature_state_table
         self._device_config = device_config
         self._cached_config = {}
         self.is_multi_npu = device_info.is_multi_npu()
+        self.enable_delayed_service = False
+        self.is_advanced_boot = is_advanced_boot
+        self.lock = threading.Lock()
+        self.port_listener_thread = threading.Thread(target=self._portListener, name='port_listener_thread')
```

Comment:
>Thread [](http://example.com/codeflow?start=46&length=6)

We discourage using threads. Could you explore if you can use `select` to implement? I understand rewrite the ConfigDB subscribe part need extra effort.

Reply:
We previously used the subscribertable approach for which we were requested to move to configdbconnector to use a single FD. We cannot use configDBconnector for this scenario as the PORT_TABLE:PortInitdone is part of APPL_DB.